src/parsers/babel_parser.js

```diff
@@ -171,3 +170,34 @@ export const parse = (file: string, data: ?string): ParseResult => {
 
   return parseResult;
 };
+
+const additionalPlugins = [
+  'jsx',
+  'asyncGenerators',
+  'bigInt',
+  'classProperties',
+  'classPrivateProperties',
+  'classPrivateMethods',
+  'doExpressions',
+  'dynamicImport',
+  'exportDefaultFrom',
+  'exportNamespaceFrom',
+  'functionBind',
+  'functionSent',
+  'importMeta',
+  'logicalAssignment',
+  'nullishCoalescingOperator',
+  'numericSeparator',
+  'objectRestSpread',
+  'optionalCatchBinding',
+  'optionalChaining',
+  'partialApplication',
+  'throwExpressions',
+  'topLevelAwait',
+];
```

Comment:
I am not sure not adding decorators and pipelineOperator is better than adding with some defaults like legacy? Either way, we should document this list in the README file and give a bit of explanation of why and the limit etc.

Reply:
i will add a list to the readme, but in general there is little information about this package in the readme. would be great if there would be a short summary about the methods this package is exposing.

Reply:
agree, let's start building up the README file

Reply:
i added some info to the Readme, please verify. actually we could let the api consumer pass the addtionalPluginList as an addtional param of the parse method? maybe this would be a better option then maintaining the list in this package?

Reply:
please check how i imagine this could work https://github.com/jest-community/jest-editor-support/pull/30/files#diff-80745ee8575f2870ead6cd5b032a42d2

this way we would not need to hardcode all the plugins into this package. instead leave it up to the API consumer what to activate. we could also still adjust which plugins we include in this package and define them in readme.md https://github.com/jest-community/jest-editor-support/pull/30/files#diff-2427e229bfc5afdbd2cbbc3ed96a650fR174

what do you think? will i get the job? :bowtie: